hw-model/src/lib.rs

```diff
@@ -792,15 +792,14 @@ pub trait HwModel {
         let Some(response_bytes) = self.mailbox_execute(R::ID.into(), req.as_bytes())? else {
             return Err(ModelError::MailboxNoResponseData);
         };
-        let response = match R::Resp::read_from(response_bytes.as_slice()) {
-            Some(response) => response,
-            None => {
-                return Err(ModelError::MailboxUnexpectedResponseLen {
-                    expected: mem::size_of::<R::Resp>() as u32,
-                    actual: response_bytes.len() as u32,
-                })
-            }
-        };
+        if response_bytes.len() > mem::size_of::<R::Resp>() {
```

Comment:
I don't think it's wise to do this for ALL commands. I think it's better to use my solution from https://github.com/chipsalliance/caliptra-sw/pull/1076, where we only do this for certain commands like GetLdevCertResp and InvokeDpeResp

Reply:
See https://github.com/chipsalliance/caliptra-sw/pull/1076/commits/f84c6b701d01ece874795a356b77bcbaf8d6c2b0

Reply:
Thanks, I have updated it with your code.